examples/proactive/src/hooks/before_incoming_middleware/contact-form.js

```diff
@@ -0,0 +1,4 @@
+if (event.type === 'contact-form') {
+  event.state.session.contactForm = event.payload.payload.payload // issh
```

Comment:
maybe we could pass sendData(payload) instead of sendData({type, payload}) ? Not sure if it would break something tho.